docs/guides/access-groups.mdx

```diff
@@ -4,27 +4,60 @@ description: How to create and manage access groups for your organization's acco
+An _access group_ contains _policies_, which define the actions that access group members can take on specific resources, such as services.  In this guide, the resource is generally an IBM Quantum _service instance_.
 
 <Admonition type="note">
 After an access group is created, you can invite users to the group by using the IBM Quantum Platform [Access management](https://quantum.cloud.ibm.com/access-groups) page in the left-hand menu.
 </Admonition>
 
-<span id="json"></span>
-## Create a Qiskit Runtime Administrators access group (JSON)
+You can create an access group by using the IBM Cloud [console,](https://cloud.ibm.com/docs/account?topic=account-groups&interface=ui) [CLI,](https://cloud.ibm.com/docs/account?topic=account-groups&interface=cli) [API,](https://cloud.ibm.com/docs/account?topic=account-groups&interface=api) or [Terraform.](https://cloud.ibm.com/docs/account?topic=account-groups&interface=terraform)
 
-After setting up an account for your organization, it's recommended that you go to the IBM Cloud console for [Access groups](https://cloud.ibm.com/iam/groups) and create a Qiskit Runtime Administrators access group. This access group gives access to other users to create and manage instances, and manage user access for the Qiskit Runtime service.
+## Create an IBM Quantum Administrators access group
+After setting up an account for your organization, it's recommended that you create an  IBM Quantum Administrators access group. This access group lets other users  create and manage instances, and manage user access for the Qiskit Runtime service.
 
-When you create this access group, include the following policies.  You can copy and paste the code into the JSON entry feature on the IAM console. Alternatively, you can make all of the selections on the user interface, as described in the [next section.](#ui)
+When you create this access group, include the following policies:
 
-- All IAM Account Management services (Administrator), required for inviting users, managing access groups and creating access policies
+- Qiskit Runtime service - Grant access to manage all IBM Quantum instances in the account and view account usage analytics.
+  - **Manager** service access role
+  - **Administrator** platform management access role
+- All account management services - Grant access to list all the resource groups in the account.
+  - **Viewer** platform management access role
+- All IAM Account Management services - Grant access to invite users, manage access groups, and create access policies.
+  - **Administrator** platform management access role
+
+
+<Admonition type="note">
+  Users with the `viewer` platform management role on "all account management services" can also view services such as billing. To prevent this extra view access, use the IBM Cloud CLI to give them access to just Resource groups:
```

Comment:
```suggestion
  Users with the `viewer` platform management role on "all account management services" can also view services such as billing. If you want to prevent this extra view access, use the IBM Cloud CLI to give them access to just Resource groups:
```